modules/simul/src/main/Env.scala

```diff
@@ -63,26 +63,25 @@ final class Env(
   def version(simulId: SimulId) =
     simulSocket.rooms.ask[SocketVersion](simulId.into(RoomId))(GetVersion.apply)
 
-  Bus.subscribeFuns(
-    "finishGame" -> { case lila.core.game.FinishGame(game, _) =>
+  Bus.sub[lila.core.game.FinishGame]:
+    case lila.core.game.FinishGame(game, _) =>
       api.finishGame(game)
-      ()
-    },
-    "adjustCheater" -> { case lila.core.mod.MarkCheater(userId, true) =>
+      () // TODO FIXME really necessary? compiles without
+
+  Bus.sub[lila.core.mod.MarkCheater]:
+    case lila.core.mod.MarkCheater(userId, true) =>
       api.ejectCheater(userId)
-      ()
-    },
-    "moveEventSimul" -> { case lila.core.round.SimulMoveEvent(move, _, opponentUserId) =>
+      () // TODO FIXME really necessary? compiles without
```

Comment:
My guess is the unit is there not to wait for the future, but since there's no return anyway it seems pointless

Reply:
Agree, I believe we can safely remove those `()`